### PR TITLE
Initializing nbdev to repository (initial PR)

### PR DESCRIPTION
**Pull request by lifeofborna**
I've just finished setting up and testing the nbdev environment for our repository with Nella. I created this early PR to make sure everything has been implemented correctly and to see if it meets the desired requirements.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,10 +36,7 @@
...
 *~
 
-
-
-#nbdev
-_docs/
+# nbdev
 _proc/
-
+_docs/
```
Comment (doyu) [inline .gitignore:42]: This doen't seem to change anything here?

File: nbs/.ipynb_checkpoints/index-checkpoint.ipynb [hunk 2]
```diff
@@ -9,6 +9,109 @@
     "#| hide\n",
     "from https://github.com/lifeofborna/tflm_hello_world/tree/main.core import *"
    ]
```
Comment (doyu) [inline nbs/.ipynb_checkpoints/index-checkpoint.ipynb]: checkpoint is a intermediate file, no need

File: README.md [hunk 3]
```diff
@@ -21,94 +21,3 @@ Fill me in please! Don’t forget code examples:
 ```
```
Comment (doyu) [inline README.md:27]: Wound't it be easier to rename README.md first and commit and then, add nbdev changes?
In this way everything would be cleaner to see.

**Comment by doyu** [comment]
There are some unnecessary changes included.
Would it be possible to create a clean PR of nbdev on the top of the current repo after you submit to rename README.md -> GUIDE.md?
It would be nice if a PR includes only necessary changes on the top of repo. Thanks!

**Review by doyu — CHANGES_REQUESTED** [review]
There are some unnecessary changes included.
Would it be possible to create a clean PR of nbdev on the top of the current repo after you submit to rename README.md -> GUIDE.md?
It would be nice if a PR includes only necessary changes on the top of repo. Thanks!

**Comment by lifeofborna** [comment]
> There are some unnecessary changes included. Would it be possible to create a clean PR of nbdev on the top of the current repo after you submit to rename README.md -> GUIDE.md? It would be nice if a PR includes only necessary changes on the top of repo. Thanks!

Hey I did what you asked but I accidentally forked it on https://github.com/Origami-TinyML/tflm_hello_world-1 and not my own fork. Sorry for that..

**Comment by lifeofborna** [comment]
Hmm, I might have misunderstood the reply then. I thought you wished me to rename the readme.md to guide.md --> commit --> push and then initialize nbdev on top of the current repo. Without the unnecessary changes?

The problem is I cannot clone our repo using SSH and then run the nbdev_new on it because it results in a JSON parsing error. Which breaks the entire nbdev for some reason. The only way to bypass that was to first download the main repo as a ZIP file then run nbdev_new on it and then merge it with the branch which results in https://github.com/Origami-TinyML/tflm_hello_world-1. Which may result in the "unnecessary changes" you mentioned in the comment.

This was the original problem I and Nella faced.

Could you clarify if I misunderstood your reply? Thanks.

**Comment by doyu** [comment]
> Hmm, I might have misunderstood the reply then. I thought you wished me to rename the readme.md to guide.md --> commit --> push and then initialize nbdev on top of the current repo. Without the unnecessary changes?

Yes, that's the change request.
>
> The problem is I cannot clone our repo using SSH and then run the nbdev_new on it because it results in a JSON parsing error. Which breaks the entire nbdev for some reason. The only way to bypass that was to first download the main repo as a ZIP file then run nbdev_new on it and then merge it with the current branch which results in https://github.com/Origami-TinyML/tflm_hello_world-1. Which may result in the "unnecessary changes" you mentioned in the comment.
> 
> This was the original problem I and Nella faced.
> 
> Could you clarify if I misunderstood your reply? Thanks.

You understand my request correctly, but at first you need to solve your git clone issue at first. Other guys have any problem with that. Just ask your problem in Discord. Someone could solve that.

After that, please send a PR for a clean repo. Thanks!

**Pull request closed without merging**